src/patternfly/components/Popover/popover.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="pf-c-overlay">
+  <div class="pf-c-overlay__container {{direction}}">
+      <div class="pf-c-popover {{direction}}">
+        <div class="pf-c-popover__arrow"></div>
+        <div class="pf-c-popover__content pf-c-modal-box" role="dialog" aria-modal="true" aria-labelledby="modal-title" aria-describedby="modal-description">
+          <div class="pf-c-modal-box__close">
+            <button class="pf-c-button pf-m-action" aria-label="Close Dialog">
+              <i class="fas fa-times"></i>
+            </button>
+          </div>
+          <header class="pf-c-modal-box__header">
+            <h1 class="pf-c-modal-box__header-title" id="modal-title">
```

Comment:
Why are we using modal classes within the popover, they are completely unrelated

Reply:
I think the original suggestion was to use a similar structure to what we have for popovers, since they're very similar (e.g. both have a header, Close action, and contents), which I think makes sense. But I agree that we shouldn't have Modal classes here.

Reply:
@andresgalante - what specifically should the developer do here?